INFO: Default loss without VGG
INFO: 2023-07-25 21:11:44 Argument batch_size: 8
INFO: 2023-07-25 21:11:44 Argument code_test: True
INFO: 2023-07-25 21:11:44 Argument data_train_path: ./../data/data_96/train/
INFO: 2023-07-25 21:11:44 Argument device_number: 3
INFO: 2023-07-25 21:11:44 Argument epoch: 1
INFO: 2023-07-25 21:11:44 Argument local_rank: 0
INFO: 2023-07-25 21:11:44 Argument max_HiC: 200
INFO: 2023-07-25 21:11:44 Argument num_gpu: 1
INFO: 2023-07-25 21:11:44 Argument num_workers: 1
INFO: 2023-07-25 21:11:44 Argument resume_epoch: 0
INFO: 2023-07-25 21:11:44 Argument resume_path: None
INFO: 2023-07-25 21:11:44 Argument rgb: False
INFO: 2023-07-25 21:11:44 Argument train_dataset: hic
INFO: 2023-07-25 21:11:44 Argument val_datasets: ['hic']
INFO: Traceback:
INFO: Traceback (most recent call last):
  File "./train_1d.py", line 297, in <module>
    train(model, args)
  File "./train_1d.py", line 167, in train
    data_gpu = data.to(device, dtype=torch.float32, non_blocking=True).astype('float32')  #B,3,C,H,W
AttributeError: 'Tensor' object has no attribute 'astype'

INFO: Sys.exc_info()[2]:
INFO: <traceback object at 0x7f8001b3ae40>
